cloud-movies-fe/src/components/all-movies/all-movies.jsx

```diff
@@ -21,10 +21,10 @@ function AllMovies() {
     <div className="all-movies">
       <Navbar />
       <div className="all-movies-content">
-        <MovieSearch />
+        <MovieSearch setMovies={setMovies} />
         <div className="all-movies-grid">
           {movies.map((movie) => (
-            <MovieCard Genre={movie.Genre} MovieId={movie.MovieId} Title={movie.Title} key={movie.MovieId} />
+            <MovieCard Genre={movie.Genre} MovieId={movie.MovieId} Title={movie.Title} CoverS3Url={movie.CoverS3Url} key={movie.MovieId} />
```

Comment:
I'm not sure whether we have cover URL on public access. If we do that might be a security problem hence videos would also be publicly accessible which we don't want.